jobs/JGLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -30,6 +30,10 @@ YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
   COMOUT_OCEAN_LETKF:COM_OCEAN_LETKF_TMPL \
   COMOUT_ICE_LETKF:COM_ICE_LETKF_TMPL
 
+RUN=${GDUMP_ENS} YMD=${PDY} HH=${cyc} MEMDIR="ensstat" declare_from_tmpl -rx \
```

Comment:
I think this should be `RUN=${RUN}` because the output is going to land in the `enkfgdas` or `enkfgfs` based on if this job is run in the early or late cycle.
```suggestion
YMD=${PDY} HH=${cyc} MEMDIR="ensstat" declare_from_tmpl -rx \
```

Reply:
Done!